src/Microsoft.AspNet.OData.Shared/Formatter/Serialization/ODataEnumSerializer.cs

```diff
@@ -89,6 +90,14 @@ public sealed override ODataValue CreateODataValue(object graph, IEdmTypeReferen
                 }
             }
 
+            // Enum member maybe support camel case. So, try to use the Edm member name to create Enum value.
+            var member = enumType.EnumDefinition().Members
```

Comment:
I think my comments maybe confusing. It's not only for camel case, it's also for model alias because it's allowed to input any string in the [EnumMember] attribute, "high", or "High", or "hIgH", etc. 

I updated my comments to refresh this point.

Reply:
Then, does the string in the [EnumMember] attribute require case-insensitive (StringComparison.OrdinalIgnoreCase)? Alias can be a completely different string from the Enum (not only case-insensitive), right?

Reply:
En… It seems you're right. It can be any valid string. Let me update the PR later.